Library/Homebrew/sandbox.rb

```diff
@@ -91,6 +91,22 @@ def deny_write_homebrew_repository
     end
   end
 
+  def allow_network(path, options = {})
```

Comment:
Change `options = {}` into a proper Ruby attributes for each option instead. Adds better type handling with Sorbet (and, ideally, type signatures should be added as much as possible).